projects/eslint-plugin/configs/all.json

```diff
@@ -4,6 +4,7 @@
         "@taiga-ui/injection-token-description": "error",
         "@taiga-ui/prefer-inject-decorator": "error",
         "@taiga-ui/prefer-self-destroy-service": "error",
+        "@taiga-ui/accessibility": "error",
```

Comment:
@nsbarsukov @vladimirpotekhin what do you think about name?

Reply:
Maybe it is better to call it directly `@taiga-ui/no-esc-keydown` ?

Reply:
fixed